src/frontend/src/lib/components/tokens/TokenCard.svelte

```diff
@@ -1,21 +1,31 @@
 <script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
 	import TokenLogo from '$lib/components/tokens/TokenLogo.svelte';
 	import TokenName from '$lib/components/tokens/TokenName.svelte';
 	import TokenSymbol from '$lib/components/tokens/TokenSymbol.svelte';
 	import Card from '$lib/components/ui/Card.svelte';
 	import { TOKEN_CARD, TOKEN_GROUP } from '$lib/constants/test-ids.constants';
+	import type { LogoSize } from '$lib/types/components';
 	import type { CardData } from '$lib/types/token-card';
 
 	export let data: CardData;
 	export let testIdPrefix: typeof TOKEN_CARD | typeof TOKEN_GROUP = TOKEN_CARD;
+	export let tokenCount: number | undefined = undefined;
+	export let logoSize: LogoSize = 'lg';
 </script>
 
 <Card noMargin testId={`${testIdPrefix}-${data.symbol}`}>
 	<TokenSymbol {data} />
 
 	<TokenName {data} slot="description" />
 
-	<TokenLogo {data} slot="icon" color="white" />
+	<TokenLogo
+		{data}
+		subLogo={nonNullish(tokenCount) ? { type: 'tokenCount', count: tokenCount } : undefined}
```

Comment:
Unrelated to this PR: just realized that `subLogo` could potentially be called `badge`.

Reply:
good one!

- PR #3084